### ISSUE TITLE
Samba Dancing glTF Model displays an error and does not appear

### ISSUE THREAD

**Issue by cx20** [CesiumGS/cesium#6433]
I tried to display the following model in Cesium.js.
http://examples.claygl.xyz/examples/basicPointLightShadow.html
https://cdn.rawgit.com/pissang/claygl/c4f45119/example/assets/models/SambaDancing/SambaDancing.gltf

However, the following error was output and could not be displayed.
http://jsdo.it/cx20/QQHP
```
Cesium.js:532 
An error occurred while rendering.  Rendering has stopped.
undefined
TypeError: Cannot read property 'parents' of undefined
TypeError: Cannot read property 'parents' of undefined
    at Nt (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:497:30928)
    at Bt (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:498:459)
    at Ae.update (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:498:19569)
    at a.update (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:492:8011)
    at tt (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:21935)
    at Ke (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:19739)
    at Qe (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:17745)
    at lt (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:25765)
    at ut (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:525:25933)
    at we.render (https://cdn.rawgit.com/cx20/gltf-test/279cba56/libs/cesium/1.44/Cesium.js:526:2226)
```

**Comment by emackey** [comment]
The crash shown in the original post here was fixed in #8756.  The SambaDancing model has a lot of errors though, and still does not display correctly in Cesium or Windows 3D Viewer.

I think the remaining problems are not Cesium's problems, so I'm closing this.  Thanks @cx20.

**Comment by cx20** [comment]
@emackey Thank you for doing a lot of research.
